frontend/src/devicelist/__tests__/DeviceList.test.jsx

```diff
@@ -88,51 +82,103 @@ test("Clicking receivers tabs renders the receiver list ", () => {
     );
   });
 
-  verifyStaticElements();
-
-  const receiverTabButton = document.getElementById("vertical-tab-1");
-  act(() => {
-    ReactTestUtils.Simulate.click(receiverTabButton);
-  });
-
+  // get to last page
+  const numClicks = Math.ceil(sampleSenders.length / 5.0) - 1; // sub 1st page already displayed
+  for (let i = 0; i < numClicks; i += 1) {
+    nextTablePage();
+  }
   const devices = document.querySelectorAll("tr.MuiTableRow-root[index]");
   expect(devices).not.toBe(null);
-  expect(devices.length).toBe(sampleReceivers.length);
-
-  sampleReceivers.forEach((receiver, index) => {
-    const rowElements = devices[index].querySelectorAll("td");
-    expect(rowElements[0].querySelector("button span svg")).not.toBe(null);
-    expect(rowElements[1].textContent).toBe(receiver.name);
-    expect(rowElements[2].textContent).toBe(receiver.serialNumber);
-    expect(rowElements[3].firstChild.textContent).toBe(receiver.status);
-    expect(rowElements[4].textContent).toBe(receiver.privateIp);
-    expect(rowElements[5].textContent).toBe(receiver.publicIp);
-    expect(rowElements[6].querySelector("button span svg")).not.toBe(null);
-  });
+  expect(devices.length).toBe(sampleSenders.length % 5);
 });
 
-test("Clicking dropdown on table row displays additional information", () => {
-  act(() => {
-    render(
-      <BrowserRouter>
-        <DeviceListPage dataSource={SampleData} />
-      </BrowserRouter>,
-      container
-    );
-  });
-
-  verifyStaticElements();
-
-  const dropdownButton = document.querySelector(
-    "tr.MuiTableRow-root[index] button"
-  );
-  act(() => {
-    ReactTestUtils.Simulate.click(dropdownButton);
-  });
-
-  const textElement = document.querySelector(
-    "tr.MuiTableRow-root td.MuiTableCell-root[colspan] h6"
-  );
-  expect(textElement).not.toBe(null);
-  expect(textElement.innerHTML).toBe("Channels");
-});
+// test("Device list renders sender page by default", () => {
+//   act(() => {
+//     render(
+//       <BrowserRouter>
+//         <DeviceListPage dataSource={SampleData} />
+//       </BrowserRouter>,
```

Comment:
we're keeping the old tests all commented because of future test-rewrite issue, right?

Reply:
Yeah just for reference on what was in the previous test and how to refactor them to be better.